reference/array/functions/array-keys.xml

```diff
@@ -8,18 +8,23 @@
  
  <refsect1 role="description">
   &reftitle.description;
-  <methodsynopsis>
+  <methodsynopsis role="procedural">
    <type>array</type><methodname>array_keys</methodname>
    <methodparam><type>array</type><parameter>array</parameter></methodparam>
-   <methodparam choice="opt"><type>mixed</type><parameter>search_value</parameter><initializer>null</initializer></methodparam>
-   <methodparam choice="opt"><type>bool</type><parameter>strict</parameter><initializer>false</initializer></methodparam>
+   </methodsynopsis>
+   <methodsynopsis role="procedural">
+   <type>array</type><methodname>array_keys</methodname>
+    <methodparam><type>array</type><parameter>array</parameter></methodparam>
+    <methodparam><type>mixed</type><parameter>search_value</parameter></methodparam>
+    <methodparam choice="opt"><type>bool</type><parameter>strict</parameter><initializer>&false;</initializer>
+   </methodparam>
```

Comment:
Correção de indentação e remoção de uma quebra de linha extra.

```suggestion
  </methodsynopsis>
  <methodsynopsis role="procedural">
   <type>array</type><methodname>array_keys</methodname>
   <methodparam><type>array</type><parameter>array</parameter></methodparam>
   <methodparam><type>mixed</type><parameter>search_value</parameter></methodparam>
   <methodparam choice="opt"><type>bool</type><parameter>strict</parameter><initializer>&false;</initializer></methodparam>
```

Reply:
Essa correção é uma espécie de commit?

Reply:
São apenas uma sugestões mesmo. mas o GitHub oferece a opção de [incorporar as sugestões](https://docs.github.com/pt/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) no seu pull request.
Como você mesmo commitou as sugestões manualmente, não será mais necessário aplicar por aqui 👍